Sources/XRPLSwift/Xrpl/Utilities/SafeDict.swift

```diff
@@ -66,8 +66,12 @@ class SafeDict<Key: Hashable, Value> {
 
     func set(key: Key, value: Value?) {
         dispatchQueue.sync {
-            self.threadUnsafeDict.removeValue(forKey: key) // added for preventing crash against below
-            self.threadUnsafeDict[key] = value
+            self.threadUnsafeDict.removeValue(forKey: key)
+
+            // added for preventing crash against below
+            if value != nil {
+                self.threadUnsafeDict.updateValue(value!, forKey: key)
+            }
```

Comment:
You can do like this.
```
if let value {
   self.threadUnsafeDict.updateValue(value, forKey: key)
}

Reply:
[09749e7](https://github.com/girin-app/XRPLSwift/pull/9/commits/09749e77015dbb714a265613e7a4e48dff75a93b)